torchci/lib/bot/mergeBot.ts

```diff
@@ -159,10 +161,14 @@ function mergeBot(app: Probot): void {
             option["classification"].replace(/['"]+/g, "")
           )
         ) {
+          const invalidClassificationMessage = option['classification'] != null ?
+          `(the classification you provided was: ${option['classification']})` :
+           "";
           await addComment(
             ctx,
-            "Revert unsuccessful: please retry the command and provide a classification, " +
-              `e.g. @pytorchbot revert -m="this breaks mac tests on trunk" -c="ghfirst".`
+            `Revert unsuccessful: please retry the command and provide a valid classification ${invalidClassificationMessage}, ` +
```

Comment:
Can we print list of valid classification here as well. Also, do not keep constant in the middle of the code, either move them to the top of the file or perhaps to separate `mergeBot.constants.ts` (which can later be included from test to avoid string duplication?)